packages/app-risk/vite.config.ts

```diff
@@ -47,7 +47,7 @@ export default defineConfig({
       formats: ['es'],
     },
     rollupOptions: {
-      external: ['react'],
+      external: ['react', 'next', 'zod', 'kysely'],
```

Comment:
```suggestion
      external: ['react', 'next', 'next/server', 'zod', 'kysely'],
```

Same as the other one.